accountEqnAct.js

```diff
@@ -707,6 +712,15 @@ class accountEqnAct extends Phaser.Scene {
 
   }
 
+    displayProfile() {
+        this.profile.alpha = 1.0;
+        this.userName = this.add.text(70,140, localStorage.getItem("playerName"), {
```

Comment:
Can we store the entire name like you are doing, but only display the first 8 characters? Long names spill into the room floor area. It looks like about 8 will fit "in the wall." 

Perhaps grab the full stored name and then pull out a substring of characters 0-7. Not sure how Javascript would do this, but there should be something.

Reply:
Sure!